electron/src/storage/types/local-repository.ts

```diff
@@ -26,6 +26,7 @@ export class LocalRepository extends RepositoryType {
         id: "?welcome",
         label: "Welcome",
         type: "Welcome"
-    }]
+    }];
 
+    updateAvailable = null;
```

Comment:
Why do we need to persist the state here?
If you update the app, it would still notify you that there is an update available.